### ISSUE TITLE
Installation Problem

### ISSUE THREAD

**Issue by galagoz** [G-Node/GinUI#90]
Hi! I've been trying to install and run GinUI on Windows 10. Installation folder at the GIN repository (https://web.gin.g-node.org/achilleas/gin-ui-installers) was downloaded and unzipped succesfullly, but Setup.exe did not run and gave the following error:

![resim](https://user-images.githubusercontent.com/34252945/50078833-46994100-01e8-11e9-9630-6412070aa1fa.png)

I was wondering if it's an actual problem or is there something wrong with my system.

**Comment by achilleas-k** [comment]
Though this is a very generic error message, my first instinct is that maybe you're running 32bit Windows? Our builds are 64bit only.

**Comment by galagoz** [comment]
Just checked again to be sure, it's 64bit W10:

![resim](https://user-images.githubusercontent.com/34252945/50079825-f079cd00-01ea-11e9-8342-e344ab9e25e9.png)

**Comment by achilleas-k** [comment]
Thanks for getting back so quickly.

Next thing to check is whether you are allowing unsigned apps to be installed. Allowing signed apps only is a security feature and is on by default in recent versions of Windows 10.

This page explains how to enable installation of unsigned apps by enabling *Developer mode*: https://pureinfotech.com/install-unsigned-windows-10-apps-powershell/
Short version: Go to **Settings > Update & Security  > For developers > Developer mode**

You may not want to disable this security feature permanently, but I am not entirely sure if the GIN UI will run after installation if you re-enable it. We do plan to get a properly signed application once we create an official release.

Let me know if this solved the issue.

**Comment by galagoz** [comment]
Thanks a lot for the detailed answer! I enabled _Developer mode_ from the Settings menu, restarted my laptop and tried to run setup.exe again.

However the problem persists for some reason. When I try to run setup.exe, first I get a warning and proceed with "Run anyway". Then, same generic error comes up again (This app can't run on your PC).

I checked other options for installing unsigned apps using PowerShell in that website, but I'm not sure if those commands are useful for this case or not.

**Comment by The-E** [comment]
Could you give us a screenshot of the warning you're getting when running the setup.exe?

**Comment by galagoz** [comment]
Here it is:

![resim](https://user-images.githubusercontent.com/34252945/50083514-5880e100-01f4-11e9-8b79-ceecf6263a6a.png)

**Comment by The-E** [comment]
A possible fix might be to turn off SmartScreen temporarily. You can do this in **Settings->Windows Security->App & browser control->Check apps and files**.

**Comment by galagoz** [comment]
I turned off SmartScreen. Now when I run setup.exe, I get the following:

![resim](https://user-images.githubusercontent.com/34252945/50084030-e3aea680-01f5-11e9-97d2-bfe408406774.png)

When I click on Run, "This app can't run on your PC" error pops up again.

**Comment by The-E** [comment]
Can you try running the .msi file by itself?

**Comment by galagoz** [comment]
When I run Setup.msi, first I get this warning again:

![resim](https://user-images.githubusercontent.com/34252945/50084817-7c462600-01f8-11e9-8cda-668ea8626ad3.png)

And then the following error:

![resim](https://user-images.githubusercontent.com/34252945/50084830-88ca7e80-01f8-11e9-9d24-c5ac22d05742.png)

**Comment by The-E** [comment]
I got no idea about this. The remaining options for why the install fails are all equally unlikely:
1. The download was somehow corrupted 
2. The Windows Installer service is not running
3. You do not have the rights to install something on this PC.

**Comment by galagoz** [comment]
Thank you very much for all the helps and for the feedback. It turns out that I've been downloading only small files and the file structure using the download button on top right at the gin-ui-installers webpage (https://web.gin.g-node.org/achilleas/gin-ui-installers). Now I downloaded Setup.exe and msi files separately and installation worked perfectly, and I was able to sign in. I should've read the warning message that pops up after clicking on the download button, sorry for the confusion.
